yarl/_url.py

```diff
@@ -1142,7 +1148,7 @@ def human_repr(self):
                     user,
                     password,
                     host,
-                    self._val.port,
+                    self.explicit_port,
```

Comment:
This line is actually covered but we currently have a problem with reporting multi-line statements